frontend/src/layouts/ProtectedLayout.tsx

```diff
@@ -1,11 +1,19 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { Navigate, Outlet } from 'react-router-dom';
 
 import { authContext } from '../hooks/useAuth';
 
 function ProtectedLayout() {
   const auth = useContext(authContext);
 
+  useEffect(() => {
+    if (auth.isAuthed && !auth.isLoading) {
+      fetch(`http://localhost:8001/match/${auth.user.username}`)
+        .then((res) => res.json())
+        .then((res) => console.log(res));
+    }
+  }, [auth]);
```

Comment:
should we unify all HTTP requests to use axios ?

Reply:
yes we should